x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/console_commands_definition.ts

```diff
@@ -523,6 +524,74 @@ export const getEndpointConsoleCommands = ({
       privileges: endpointPrivileges,
     }),
   });
+  if (crowdstrikeRunScriptEnabled) {
+    consoleCommands.push({
+      name: 'runscript',
+      about: getCommandAboutInfo({
+        aboutInfo: CROWDSTRIKE_CONSOLE_COMMANDS.runscript.about,
+        isSupported: doesEndpointSupportCommand('runscript'),
+      }),
+      RenderComponent: () => null,
+      meta: {
+        agentType,
+        endpointId: endpointAgentId,
+        capabilities: endpointCapabilities,
+        privileges: endpointPrivileges,
+      },
+      exampleUsage: `C:\\> runscript -CloudFile="CloudScript1.ps1" -CommandLine="-Verbose true"
```

Comment:
can we shorten this? IMO, Its too much content to display under the console's command input area. That areas is really meant to give the user a quick "tip" on command usage. I would suggest just picking one of the arguments and show that as the example usage.

The output for `runscript --help` should then provide much more detail if you like.

Reply:
Absolutely, will shorten :)